### DISCUSSION TITLE
k8s 部署，启动后报错 error:Read-only file system

### DISCUSSION THREAD

**Discussion by hanzoi** [alibaba/ilogtail#484] [Help] [answered]
背景：集群为ACK容器集群，需要ds方式部署ilogtail，采集pod中文件日志并输出到阿里云kafka中。将日志投递到阿里云kafka需要配置 Version 字段，故自行clone后，make dist make docker 构建镜像。

配置完成后启动Pod出现如下报错：
/src/core/config_manager/ConfigManagerBase.cpp:2427             create conf yaml dir failed:/usr/local/ilogtail/./user_yaml_config.d/remote_config/     error:Read-only file system

<img width="1507" alt="企业微信截图_b9a24a14-5a72-4e7b-9822-18b7bd3833c5" src="https://user-images.githubusercontent.com/61114924/197463090-811f7f08-081b-4ef0-abbb-06bc48ad4d24.png">

**Comment by shalousun** [comment]
@hanzoi 可以贴下你的部署模板，可能模板configmap挂载权限存在问题，社区有其他同学编译flusher kafkav2使用是ok的。

**Reply by hanzoi** [reply]
这是 configmap 配置
<img width="1234" alt="image" src="https://user-images.githubusercontent.com/61114924/197722972-b1e6e182-5668-43bf-be22-4aa0db47915f.png">

ds 的 yaml 是把文档里的直接复制过去的
https://ilogtail.gitbook.io/ilogtail-docs/v/pre-release/installation/start-with-k8s

**Comment by henryzhx8** [comment, marked as answer]
user_yaml_config.d被configmap挂载了，所以没法往里再新建目录（用于config server的目录），后续要调整一下目录组织形式

**Comment by henryzhx8** [comment]
现在先把这块代码注释掉了， #491
